components/eam/src/physics/cam/hetfrz_classnuc_cam.F90

```diff
@@ -133,15 +126,13 @@ module hetfrz_classnuc_cam
 integer, allocatable :: mode_idx(:)
 integer, allocatable :: spec_idx(:)
 
-! Copy of cloud borne aerosols before modification by droplet nucleation
```

Comment:
We can probably clean up `MAM3_nmodes` and `MAM7_nmodes` on lines 69 and 70 (original code line numbers)